examples/window-app/efr32/BUILD.gn

```
@@ -84,13 +90,25 @@ if (silabs_board == "BRD4166A" || silabs_board == "BRD2601B" ||
   disable_lcd = true
 }
 
+defines = []
+
 # WiFi settings
 if (chip_enable_wifi) {
   # disabling LCD for MG24 for wifi
   if (silabs_board == "BRD4186C" || silabs_board == "BRD4187C") {
     show_qr_code = false
     disable_lcd = true
   }
+  if (ssid != "") {
+    defines += [
+      "CHIP_ONNETWORK_PAIRING = 1",
+      "CHIP_WIFI_SSID = $ssid",
+    ]
+  }
+  if (psk != "") {
+    assert(ssid != "", "ssid can't be null if psk is provided")
+    defines += [ "CHIP_WIFI_PSK = $psk" ]
```

Seems like we want:

```
defines += [ "CHIP_WIFI_PSH=\"${psk}\"" ]
```
```suggestion
    defines += [ "CHIP_WIFI_PSK = $psk" ]
```

Same everywhere else: make the defines a string if it needs to be a string rather than just text.